argostime/crawler/ekoplaza.py

```diff
@@ -0,0 +1,81 @@
+#!/usr/bin/env python3
+"""
+    crawler/ekoplaza.py
+
+    Crawler for ekoplaza.nl
+
+    Copyright (c) 2022 Anna <anna [at] anna.computer>
+
+    This file is part of Argostimè.
+
+    Argostimè is free software: you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation, either version 3 of the License, or
+    (at your option) any later version.
+
+    Argostimè is distributed in the hope that it will be useful,
+    but WITHOUT ANY WARRANTY; without even the implied warranty of
+    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+    GNU General Public License for more details.
+
+    You should have received a copy of the GNU General Public License
+    along with Argostimè. If not, see <https://www.gnu.org/licenses/>.
+"""
+
+
+import json
+import logging
+from pprint import pprint
+
+
+import requests
+
+from argostime.exceptions import CrawlerException
+from argostime.exceptions import PageNotFoundException
+from argostime.crawler.crawl_utils import CrawlResult
+
+
+def crawl_ekoplaza(url: str):  # -> CrawlResult:
+    """Ekoplaza crawler"""
+
+    info = url.split('product/')[-1]
+    response = requests.get(
+        f'https://www.ekoplaza.nl/api/aspos/products/url/{info}')
```

Comment:
Interessant dat er gewoon een API achter zit, maakt het ons wel een stukje makkelijker :)

Reply:
Er zit ook veel te veel info in die API, bijv. 'PurchasePrice' wat de inkoopprijs voor de supermarkt lijkt te zijn

Reply:
Hah, wauw